### ISSUE TITLE
Support Jetbrains' remote development server-to-client flow (remove code with me relay)

### ISSUE THREAD

**Issue by atduarte** [gitpod-io/gitpod#5644]
Without using _code with me_ servers as a relay.

See https://youtrack.jetbrains.com/issue/IDEA-226455#focus=Comments-27-5125731.0-0

We will use HTTP links that trigger a Custom Connector in the local Gateway/IDE. Details pending. [[1](https://gitpod.slack.com/archives/C02BRJLGPGF/p1633951005117700?thread_ts=1633596972.116200&cid=C02BRJLGPGF)]

**Comment by loujaybee** [comment]
Adding the blocked label to this one ([internal link](https://gitpod.slack.com/archives/C02CBKZEVS8/p1635848425045900?thread_ts=1635847455.044200&cid=C02CBKZEVS8)) as we're still waiting on JetBrains to come back to us regarding the fix/solution for this.

**Comment by loujaybee** [comment]
See also [this discussion with JetBrains](https://gitpod.slack.com/archives/C02BRJLGPGF/p1635850770016300). Their [documentation can be found here](https://docs.google.com/document/d/1ioL_6obENQHdUQ8RTipPL9wf3vz6WjzlGSxPFVhCcg0/edit#heading=h.lukfxhxjuybw).

**Comment by loujaybee** [comment]
Going to close as duplicate of wrapper epic: https://github.com/gitpod-io/gitpod/issues/7183
